### PR TITLE
Add Flutter LICENSE and headers

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

**Comment by zanderso** [comment]
cc @bdero

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2022 Brandon DeRosier. All rights reserved.
+Copyright 2013 The Flutter Authors. All rights reserved.
```
Comment (bdero) [inline LICENSE:1]: Does the year matter?

Comment (zanderso) [reply LICENSE:1]: 2013 is correct.

**Review by bdero — APPROVED** [review]
LGTM

**Pull request merged**